hunts-api/routes/alive.js

```diff
@@ -2,16 +2,25 @@ const router = require('express').Router();
 const getDb = require('../db');
 
 
-router.get('/', (req, res) => {
-  getDb().then(db => {
+ router.get('/', (req, res) => {
 
-    // db is available here just like in:
+  getDb().then(db => {
+    // db is available ere just like in:
     // http://mongodb.github.io/node-mongodb-native/3.0/quick-start/quick-start/#insert-a-document
+      db.collection('hunts')
+      .find({})
+      .toArray((err, results) =>{
+        if(err)throw err;
+        console.log("Documents found are: ", results);
```

Comment:
`res.json` should go inside of this callback

you should `res.status(500).json(err)` instead of `throw err`